test/Makefile

```diff
@@ -193,6 +193,18 @@ testrunc99/% : $(SMALL1)/%.c
 	cd $(SMALL1); ./$*.exe
 	echo SUCCESS
 
+# TODO: how to make this just run cil, not gcc?
```

Comment:
Does basing it off the normal `test/%` not help here?

Reply:
I'm not sure anymore why I wanted this, maybe for some of the TODO tests that are commented out?

Also, it is based on `test/%`, just with a different standard. Both call cilly, which runs gcc I think.